letsencrypt/Dockerfile

```diff
@@ -36,8 +35,9 @@ RUN \
         musl-dev \
         openssl-dev \
         cargo \
+        py3-pip \
```

Comment:
Why is this suddenly necessary? Since we already have add-on deployed, pip definitely worked there.

Reply:
See above comment as it is related

Reply:
I will remove it